src/applications/ivc-champva/shared/components/fileUploads/MissingFileOverview.jsx

```diff
@@ -64,13 +57,6 @@ const mailInfo = (
           <p>Mail your {appType} and supporting document copies to:</p>
         </>
       )}
-      {showOpt ? (
-        <p>
-          Optional files are not required to complete your {appType}, but may
-          prevent delays in your processing time.
-          <br />
-        </p>
-      ) : null}
```

Comment:
Since we no longer care about displaying optional files in this component, this is not needed.